packages/api/src/schema/types/clinvar.js

```diff
@@ -9,81 +10,62 @@ import { fetchAllSearchResults } from '../../utilities/elasticsearch'
+    main_transcript_lof: { type: GraphQLString },
+    main_transcript_lof_filter: { type: GraphQLString },
+    main_transcript_lof_flags: { type: GraphQLString },
+    main_transcript_lof_info: { type: GraphQLString },
+    main_transcript_major_consequence: { type: GraphQLString },
+    main_transcript_major_consequence_rank: { type: GraphQLInt },
+    main_transcript_protein_id: { type: GraphQLString },
+    main_transcript_transcript_id: { type: GraphQLString },
     pos: { type: GraphQLFloat },
-    xpos: { type: GraphQLFloat },
     ref: { type: GraphQLString },
-    alt: { type: GraphQLString },
-    variant_id: { type: GraphQLString },
-    measureset_type: { type: GraphQLString },
-    measureset_id: { type: GraphQLString },
-    allele_id: { type: GraphQLString },
-    symbol: { type: GraphQLString },
-    hgvsc: { type: GraphQLString },
-    hgvsp: { type: GraphQLString },
-    molecular_consequence: { type: GraphQLString },
-    clinical_significance: { type: GraphQLString },
-    pathogenic: { type: GraphQLString },
-    benign: { type: GraphQLString },
-    inflicted: { type: GraphQLString },
     review_status: { type: GraphQLString },
-    gold_stars: { type: GraphQLString },
-    all_submitters: { type: GraphQLString },
-    all_traits: { type: GraphQLString },
-    all_pmids: { type: GraphQLString },
-    inheritance_modes: { type: GraphQLString },
-    age_of_onset: { type: GraphQLString },
-    prevalence: { type: GraphQLString },
-    disease_mechanism: { type: GraphQLString },
-    origin: { type: GraphQLString },
-  }),
+    transcript_consequence_terms: { type: GraphQLString },
+    transcript_ids: { type: GraphQLString },
+    variant_id: { type: GraphQLString },
+    xpos: { type: GraphQLFloat },
+  },
 })
 
 export default clinvarType
 
 
-export async function lookupClinvarVariantsByGeneName(client, geneName) {
-  const fields = [
-    'chrom',
-    'pos',
-    'ref',
-    'alt',
-    'MEASURESET_TYPE',
-    'MEASURESET_ID',
-    'ALLELE_ID',
-    'SYMBOL',
-    'HGVS_C',
-    'HGVS_P',
-    'MOLECULAR_CONSEQUENCE',
-    'CLINICAL_SIGNIFICANCE',
-    'PATHOGENIC',
-    'BENIGN',
-    'CONFLICTED',
-    'REVIEW_STATUS',
-    'GOLD_STARS',
-    'ALL_SUBMITTERS',
-    'ALL_TRAITS',
-    'ALL_PMIDS',
-    'INHERITANCE_MODES',
-    'AGE_OF_ONSET',
-    'PREVALENCE',
-    'DISEASE_MECHANISM',
-    'ORIGIN',
-  ]
-
-  const esVariants = await fetchAllSearchResults(
+export function lookupClinvarVariantsByGeneId(client, geneId) {
+  return fetchAllSearchResults(
     client,
     {
-      index: 'clinvar',
+      index: 'clinvar_grch37',
       type: 'variant',
-      _source: fields,
       body: {
         query: {
           bool: {
-            must: [
-              { term: { SYMBOL: geneName } },
-              { exists: { field: 'HGVS_P' } },
+            filter: [
+              { term: { gene_ids: geneId } },
+              { exists: { field: 'main_transcript_hgvsp' } },
```

Comment:
I forget, why is this filter applied?

Reply:
I don't know. It's been there since ClinVar variants were added to the API in https://github.com/macarthur-lab/gnomadjs/commit/e41e641e9f04420573c3e467620200d8f2c9d867.

Reply:
May want to try removing it -- it could be filtering splice variants.